### PR TITLE
[PM-5009] migrate billing component

### PR DESCRIPTION
**Pull request by vinith-kovan**
## Type of change

```
- [ ] Bug fix
- [ ] New feature development
- [ ] Tech debt (refactoring, code cleanup, dependency upgrades, etc)
- [ ] Build/deploy pipeline (DevOps)
- [x] Other
```

## Objective
Migrating billing component css to Tailwind css

## Code changes
billing.component.html

## Screenshots
After Migration
image

Before Migration
image

### CODE REVIEW

File: apps/web/src/app/billing/accounts/trial-initiation/billing.component.html [hunk 1]
```diff
@@ -1,73 +1,72 @@
...
+          </bit-label>
+        </bit-radio-button>
+      </bit-radio-group>
     </div>
 
     <div class="tw-mb-4">
-      <h2 class="tw-mb-3 tw-text-base tw-font-semibold">{{ "paymentType" | i18n }}</h2>
+      <h2 bitTypography="h2" class="tw-mb-3 tw-text-base tw-font-semibold">
+        {{ "paymentType" | i18n }}
+      </h2>
```
Comment (willmartian) [inline apps/web/src/app/billing/accounts/trial-initiation/billing.component.html:64]: Ideally this should be a fieldset legend, that is more semantically appropriate than a heading. 

@vinith-kovan could you revert this change for now?

@bitwarden/team-billing-dev should this heading be part of the `app-payment` component itself?

File: apps/web/src/app/billing/accounts/trial-initiation/billing.component.html [hunk 2]
```diff
@@ -1,73 +1,72 @@
...
       <app-payment [hideCredit]="true" [trialFlow]="true"></app-payment>
       <app-tax-info [trialFlow]="true" (onCountryChanged)="changedCountry()"></app-tax-info>
     </div>
 
-    <div class="tw-flex tw-space-x-2">
+    <div class="tw-flex tw-gap-x-2">
```
Comment (willmartian) [inline apps/web/src/app/billing/accounts/trial-initiation/billing.component.html:69]: FYI for anyone else who sees this change, `tw-gap` does the same thing as `tw-space` here. `tw-space` approximates the behavior for older browsers that don't support the flexbox gap property, which doesn't apply to our target browser range.

**Review by willmartian — CHANGES_REQUESTED** [review]

File: apps/web/src/app/billing/accounts/trial-initiation/billing.component.html (hunk 1 above)
Comment (vinith-kovan) [reply apps/web/src/app/billing/accounts/trial-initiation/billing.component.html:64]: Sure

File: apps/web/src/app/billing/accounts/trial-initiation/billing.component.html [hunk 3]
```diff
@@ -1,73 +1,72 @@
...
+      <bit-radio-group formControlName="plan" [block]="true">
+        <bit-label>{{ "billingPlanLabel" | i18n }}</bit-label>
+        <bit-radio-button
+          *ngFor="let selectablePlan of selectablePlans"
+          id="interval{{ selectablePlan.type }}"
+          [value]="selectablePlan.type"
+        >
+          <bit-label>
+            <span *ngIf="selectablePlan.isAnnual">
+              <span *ngIf="selectablePlan.PasswordManager">
```
[the remark this answers is not in the document]
Comment (vinith-kovan) [reply apps/web/src/app/billing/accounts/trial-initiation/billing.component.html:13]: Actually, it is not necessary i will revert it to ng-container

File: apps/web/src/app/billing/accounts/trial-initiation/billing.component.html (hunk 1 above)
Comment (amorask-bitwarden) [reply apps/web/src/app/billing/accounts/trial-initiation/billing.component.html]: @willmartian Are you referring to this `<h2>` above?

**Review by willmartian — APPROVED** [review]
LGTM 👍

**Review by amorask-bitwarden — APPROVED** [review]

**Comment by vinith-kovan** [comment]
@amorask-bitwarden, while resolving conflicts, it seems that billing-component.html got deleted so we don't need this PR.

**Comment by amorask-bitwarden** [comment]
> @amorask-bitwarden, while resolving conflicts, it seems that billing-component.html got deleted so we don't need this PR.

@vinith-kovan Yeah we can close this, thank you for checking.

**Comment by cturnbull-bitwarden** [comment]
Closing this PR as it's no longer needed. Thank you for your work on it, @vinith-kovan!

**Pull request closed without merging**